crates/typos-dict/assets/words.csv

```diff
@@ -146,6 +147,7 @@ absailing,abseiling
 absance,absence
 abscence,absence
 abscound,abscond
+absed,based
```

Comment:
Can you update this to `absed,abased,based`

Reply:
Updated at [cb37366](https://github.com/crate-ci/typos/pull/530/commits/cb3736663e61ab7d97839481696bae0c77434865)